packages/sdk-metrics/test/MeterProvider.test.ts

```diff
@@ -463,6 +463,69 @@ describe('MeterProvider', () => {
         }
       );
     });
+
+    it('with instrument unit should apply view to only the selected instrument unit', async () => {
+      // Add view that renames 'test-counter-ms' and 'test-counter-s' to 'renamed-instrument' on 'meter1'
```

Comment:
Possibly a test that uses Histograms instead would be better. Instead of renaming them, we could change its `aggregation` to an `ExplicitBucketHistogramAggregation` with different bucket boundaries based on the unit. You could then assert the bucket boundaries at `(resourceMetrics.scopeMetrics[0].metrics[0] as HistogramMetricData).dataPoints[0].value.buckets.boundaries` to make sure they're correct. :thinking: 

Per spec, `View`s allow for bulk-changing the aggregation of instruments, as long as the `View` does not rename. I think the suggested test above would be more akin to the intended use-case of the `instrumentUnit` selector. :slightly_smiling_face:

Reply:
Done :heavy_check_mark: